js/views/modals/moderatorDetails.js

```diff
@@ -36,13 +38,21 @@ export default class extends BaseModal {
   }
 
   render() {
+    const modLanguages = [];
+
+    this.model.get('moderatorInfo').get('languages').forEach(lang => {
+      const langName = _.findWhere(languages, { code: lang }).name;
```

Comment:
The languages module has a `getLangByCode()` function which utilizes a cached indexed list of langs, so it would likely be more efficient than the `findWhere` on each iteration.

Also, in case we don't have a langName, I would think we would just want to display the code. It's possible some other client allowed in a lang code we don't recognize.

```
const langData = getLangByCode(lang);
modLanguages.push(langData && langData.name || lang);
```